rocketmq-flink/src/main/java/org/apache/rocketmq/flink/RocketMQSink.java

```diff
@@ -152,21 +166,36 @@ public void onException(Throwable throwable) {
     }
 
     private Message prepareMessage(IN input) {
-        String topic = topicSelector.getTopic(input);
-        String tag = (tag = topicSelector.getTag(input)) != null ? tag : "";
+        Validate.notNull(input, "the input is null");
+
+        if (keyValueSerializationSchema != null) {
```

Comment:
Is it necessary to judge not null here? since you have validated it in the open method.